source/fem-dem/vans_assemblers.cc

```diff
@@ -2236,3 +2236,63 @@ VANSAssemblerFPI<dim>::assemble_rhs(
 }
 template class VANSAssemblerFPI<2>;
 template class VANSAssemblerFPI<3>;
+
+template <int dim>
+void
+VANSAssemblerFPIProj<dim>::assemble_matrix(
+  const NavierStokesScratchData<dim>   &scratch_data,
+  StabilizedMethodsTensorCopyData<dim> &copy_data)
```

Comment:
```suggestion
  [[maybe_unused]] const NavierStokesScratchData<dim>   &scratch_data,
  [[maybe_unused]] StabilizedMethodsTensorCopyData<dim> &copy_data)
```

In C++ 20 we use the maybe_unused attribute in this case instead of having the (void) like you put below. It's not a big deal, but it is worth it to get the good habits now :)